### PR TITLE
Create footer

### PR DESCRIPTION
**Pull request by qjack001**
Here's my thought

### CODE REVIEW

**Comment by HoloPollock** [comment]
image

File: src/components/Film.vue [hunk 1]
```diff
@@ -18,6 +18,10 @@
 			<p>of total films {{ total }}</p>
 			<p>Percentage of films directed by women: <strong>{{ percentage }}</strong></p>
 		</div>
+		<div v-if="done" class="output" id="footer">
+			<p>This site is a project of <a href="https://goodbyte.ca">Goodbyte</strong> &copy;2020</p>
```
Comment (HoloPollock) [inline src/components/Film.vue]: end you a tag will compile

Comment (qjack001) [reply src/components/Film.vue]: oh lol

**Comment by HoloPollock** [comment]
just fyi beucase goodbye.ca is a redirect it needs to be http

**Comment by qjack001** [comment]
ok all done. @HoloPollock now we need to decide which to use

**Comment by HoloPollock** [comment]
I think yours is fine just the only question would be do we want to include a link to the support thing (honestly not a huge deal no one ever clicks it anyways lol

**Comment by qjack001** [comment]
> I think yours is fine just the only question would be do we want to include a link to the support thing (honestly not a huge deal no one ever clicks it anyways lol

Ya i figured it would be better not to in this case.

**Review by HoloPollock — APPROVED** [review]

**Comment by HoloPollock** [comment]
cool lets just merge yours then

**Pull request merged**